sql/dql/basic-queries/where-3.md

```diff
@@ -32,36 +32,65 @@ aspects:
 
 ### `LIKE`, `IN`
 
-In a `WHERE` clause, we can also use the keywords `LIKE` and `IN`. `LIKE` allows us to define a pattern that the given attribute should conform to, such as the length or the first letter. `IN` allows us to define a list of values of which the attribute must be one. See the following examples:
+In a `WHERE` clause, we can also use the keywords `LIKE` and `IN`. `LIKE` allows us to define a pattern that the given attribute should conform to, such as the length or the first letter. `IN` allows us to define a list of values of which the attribute must be one. 
```

Comment:
a better introduction for this would be 

> SQL provides a bunch of special operators to help you write complex querying conditions. More specifically, in this insight, we will take a look at:
> - the `LIKE` operator which is used for pattern matching ...
> - the `IN` ...
> - the `BETWEEN` ...